src/Logging.AzureAppServices/src/AzureAppServicesLoggerFactoryExtensions.cs

```diff
@@ -21,15 +21,17 @@ public static class AzureAppServicesLoggerFactoryExtensions
         /// Adds an Azure Web Apps diagnostics logger.
         /// </summary>
         /// <param name="builder">The extension method argument</param>
-        public static ILoggingBuilder AddAzureWebAppDiagnostics(this ILoggingBuilder builder)
+        /// <param name="customPrefix"></param>
+        /// <returns></returns>
+        public static ILoggingBuilder AddAzureWebAppDiagnostics(this ILoggingBuilder builder, string customPrefix = null)
```

Comment:
@Pilchie is this a binary breaking change?

Reply:
Absolutely.

Reply:
Overloads are binary breaking safe, optional parameters are not - think about how they work, the method always receives two arguments, it's just that if you don't specify it, the compiler burns in the default value at the call-site.

Reply:
My bad, I removed the optional parm and went ahead and created an overload instead. But CI still shows this optional parm error, even though I renamed the parameter.
@Pilchie @davidfowl  
[SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters"